content/en/docs/concepts/scheduling-eviction/taint-and-toleration.md

```diff
@@ -220,7 +220,7 @@ are true. The following taints are built in:
     as unusable. After a controller from the cloud-controller-manager initializes
     this node, the kubelet removes this taint.
 
-In case a node is to be evicted, the node controller or the kubelet adds relevant taints
+In case a Pod is to be evicted, the node controller or the kubelet adds relevant taints
```

Comment:
This paragraph actually talks about nodes, but evicted is probably not the right verb here. This should be "If a node is to be drained, ...."

Reply:
So using the same intro or phrase - in case, I would edit it to "In case a node is to be drained,"